packages/http-specs/specs/payload/pageable/main.tsp

```diff
@@ -52,4 +52,168 @@ namespace ServerDrivenPagination {
 
     @nextLink next?: url;
   };
+
+  @route("/continuationtoken")
+  namespace ContinuationToken {
+    @scenario
+    @scenarioDoc("""
+      Test case for using continuation token as pagination. Continuation token is passed in the request query and response body.
+      
+      Two requests need to be tested.
+      1. Initial request:
+      Expected route: /payload/pageable/server-driven-pagination/continuationtoken/request-query-response-body
+      Expected response body:
+      ```json
+      { "pets": [
+          { "id": "1", "name": "dog" },
+          { "id": "2", "name": "cat" }
+        ],
+        "nextToken": "12345"
```

Comment:
nit: more reasonable string?

Reply:
Agree on more meaningful token value. We may do extra test on the case of list from the 2nd page.

Reply:
@tadelesh / @weidongxu-microsoft more meaningful value means `second-page` or just `2` or other values. Appreciate if you could share one.

Reply:
To me, e.g. "page2" would do (at least better than 12345)

Reply:
Updated